plugin/build.gradle

```diff
@@ -198,7 +202,7 @@ testClusters.integTest {
     // When running integration tests it doesn't forward the --debug-jvm to the cluster anymore
     // i.e. we have to use a custom property to flag when we want to debug elasticsearch JVM
     // since we also support multi node integration tests we increase debugPort per node
-    if (System.getProperty("opensearch.debug") != null) {
+    if (System.getProperty("cluster.debug") != null) {
```

Comment:
have you tried with option `-debug-jvm` that should work also.

Reply:
Yeah, I did try that. But unfortunately it doesn't seem to be passed to the actual cluster being spun by the integ test.
So the test itself is debuggable but not the cluster spun by it.